proto/toit/api/user.proto

```diff
@@ -121,7 +121,8 @@ message UserChange {
     bool set_settings_welcome_closed = 1;
     bool set_settings_newsletter = 2;
     string role = 3;
-    string name = 4; 
+    string name = 4;
+    google.protobuf.Timestamp questionnaire_answered_at = 5;
```

Comment:
let it be an empty struct that is set. 
We want the timestamp to be controlled from server side.

Reply:
use `google.protobuf.Empty`